nx.json

```diff
@@ -28,10 +28,10 @@
       "dependsOn": ["^build"]
```

Comment:
Let's keep these dependencies for now. After the changes to the bundles are merged they will no longer be needed.